### DISCUSSION TITLE
Set default configer  UpdateCheckMessageShown

### DISCUSSION THREAD

**Discussion by TheKozi** [keepassxreboot/keepassxc#7884] [Q&A]
How i can set the default configer for "UpdateCheckMessageShown" when i deploy keepassxc?
default is set "UpdateCheckMessageShown=true"

The only was i found is this.
%appdata%\KeePassXC\keepassxc.ini

[General]
ConfigVersion=2
UpdateCheckMessageShown=true

**Comment by droidmonkey** [comment]
That's how you do it, there was another issue asking the same thing and person just did that
